### DISCUSSION TITLE
Sent & Forwarded Items

### DISCUSSION THREAD

**Discussion by MalcCurlew** [bkaankose/Wino-Mail#433] [Q&A] [answered]
Help, first time ever in this forum or GitHub come to that so please be gentle.
How do I change sent items to display the recipient rather than the sender which is always me?
Also can forwarded items either be marked accordingly or be in the sent items folder?

**Comment by bkaankose** [comment]
Hello and welcome to GitHub.

All folders will display "From" avatars when listing. Since you are the 'From' part of the mail for Sent Items, it will show you. App does not interfere with this behavior.

Regarding the forwarded items; it's up to your mail server. Wino does not move them automatically to sent folder but just synchronizes where your mail server moved that item.

**Comment by MalcCurlew** [comment]
Hi,

Many thanks for your very timly respoce, much appreciated.

I thought that this app was meant to closly mimic the windows mail app but both of these isues are not the same.

My windows mail app shows the recipiant in the sent items folder see screenshot 1.

It also identifies forwarded items with an arrow, see screenshot 2.

In my limited use so far of Wino, it would be better if both these items were as the original windows mail, is there any way I can adjust it to do so?


Sent from Wino Mail for Windows

________________________________

**Comment by bkaankose** [comment, marked as answer]
I will take these as an enhancements into backlog. If Windows Mail works like that, Wino should work the same.

**Reply by MalcCurlew** [reply]
Any Idea when these enhancements will be implemented? Its very annoying not being able to see who you sent an email too.

**Reply by 4x4offroadworld-dev** [reply]
I agree. New to the app. Was there ever a resolution?

**Comment by MalcCurlew** [comment]
Hi Burak,

Just installed latest version, this still doesn't work properly


Sent from Wino Mail<https://github.com/bkaankose/Wino-Mail/> for Windows


________________________________

**Reply by bkaankose** [reply]
Hi, unfortunately this is not yet implemented. I may include it in the next releases.

**Comment by tribal666** [comment]
Hello,

I use Winomail and I have the same request. In the “Sent Items” folder, I would like the recipient’s address to be displayed instead of my own email address.

Do you know whether this could be changed in the near future?

It is difficult to check an email sent to someone when it is not possible to search for the recipient’s email address in the list.

Many thks

**Comment by MalcCurlew** [comment]
Hi,

I totally agree, should have been that way from the get go IMO.

I have stopped using Wino because of this and other issues.

I have no idea if or when it will be corrected.
